models.py

```diff
@@ -22,3 +25,15 @@ def forward(self, x):
         x = x.view(-1, 784)
         x = self.layers(x)
         return F.log_softmax(x, dim=1)
+
+class HuggingFaceModel:
```

Comment:
Does this need to inherit from torch.nn.Module for the `forward` override to work?

Reply:
Switched to `__call__` instead of forward